tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/InterleavedDeclSuite.scala

```diff
@@ -112,47 +112,4 @@ class InterleavedDeclSuite extends BaseDottySuite {
       )
     }
   }
-
-  test("single ..., without tparams") {
```

Comment:
Moved to DottySuccessSuite in Scala 2, since it uses quasiquotes which are not yet supported